backend/src/main/java/org/sefglobal/core/repository/AmbassadorRepository.java

```diff
@@ -10,4 +12,5 @@
 public interface AmbassadorRepository extends JpaRepository<Ambassador, Long> {
     Ambassador findByIdAndStatus(long id, String status);
     List<Ambassador> findAllByStatus(String status);
```

Comment:
Make this also a pagable method

Reply:
This method is using for an internal purpose. (to generate links) Since this isn't exposed to the users shall we create this as it is for now?

Reply:
ack